test/functional/project.js

```diff
@@ -95,16 +98,19 @@ describe('Project model', () => {
         });
     });
 
-    it('omits expired projects', () => {
```

Comment:
this test doesn't really make sense any more because there is no seed with `{title: 'expired research', status: 'inactive' }`.

All searches have always been bound to a specific status, and an inactive project wouldn't have an expiry date anyway.